libs/iavl/mutable_tree.go

```diff
@@ -213,6 +213,10 @@ func (tree *MutableTree) Get(key []byte) []byte {
 	if tree.root == nil {
 		return nil
 	}
+	if getForceReadIavl() {
+		_, value := tree.ImmutableTree.GetWithIndex(key)
+		return value
+	}
 
```

Comment:
I think it might be better to write it like this
```go
if !getForceReadIavl() {
    if value, ok := tree.fastGetFromChanges(key); ok {
		return value
    }
}
```
and why ImmutableTree does not need to be changed?

Reply:
I find the immutableTree's Get is only used by MutableTree's Get,so get from iavl from mutableTree directly